keycastr/KCKeyboardTap.m

```diff
@@ -87,7 +87,20 @@ -(BOOL) installTapWithError:(NSError **)error {
     if (tapInstalled) {
         return YES;
     }
-    
+
+    // 10.9 and after
+    if (AXIsProcessTrustedWithOptions != NULL) {
```

Comment:
10.9 is our minimum supported version in the project file, so this check probably isn't needed. Also, this style generates a warning